crates/astria-sequencer/src/service/info/mod.rs

```diff
@@ -70,11 +67,27 @@ impl Info {
     async fn handle_info_request(self, request: InfoRequest) -> Result<InfoResponse, BoxError> {
         match request {
             InfoRequest::Info(_) => {
+                let block_height = self
+                    .storage
+                    .latest_snapshot()
+                    .get_block_height()
+                    .await
+                    .unwrap_or(0);
+                let app_hash = crate::app_hash::AppHash::from(
+                    self.storage
+                        .latest_snapshot()
+                        .root_hash()
+                        .await
+                        .context("failed to get app hash")?,
+                );
+
                 let response = InfoResponse::Info(response::Info {
                     version: "0.1.0".to_string(),
```

Comment:
You can use `env!("CARGO_PKG_VERSION")` to compile the version of sequencer into this response. See here for available cargo env vars: https://doc.rust-lang.org/cargo/reference/environment-variables.html

You can make use of the https://crates.io/crates/vergen crate to also grab the sha commit, which would be even more useful (because our versions right now are pretty meaningless).

Reply:
nice, will do!